js/src/annotations/osd-region-draw-tool.js

```diff
@@ -162,6 +164,9 @@
       };
       var hoverColor = this.state.getStateProperty('drawingToolsSettings').hoverColor;
       var annotations = [];
+      if (this.horizontallyFlipped) {
+        location.x = this.svgOverlay.viewer.tileSources.width - location.x;
```

Comment:
Here and one other place, we will need to go back after merging image choice to use the canvas coordinate system instead of the tilesources.